src/screens/TypeScreen.tsx

```diff
@@ -43,12 +43,13 @@ export const TypeScreen: React.FC<TypeScreenProps> = ({ navigation }) => {
   const [account, setAccount] = useState(user.name || '')
 
   const handleAddSecretButtonPress = async () => {
-    await add({ uid: user.uid, secret, account, issuer })
+    const cleanedSecret = secret.replace(/\s/g, '').toUpperCase()
+    await add({ uid: user.uid, secret: cleanedSecret, account, issuer })
     navigation.navigate('Home')
   }
 
   const invalidSecret = !RFC4648_REGEX.test(
-    secret.trim().replace(/ /g, '').toUpperCase()
+    secret.replace(/\s/g, '').toUpperCase()
```

Comment:
why not use `cleanedSecret` here?

Reply:
Because it is out of scope, as it is defined in the `handleAddSecretButtonPress`
You could always extract it at the same level of the `useState` and reuse it everywhere

Reply:
just because `cleanedSecred` is not in the same scope. 
I think we can achieve something more efficient here, let me try something 😉